src/Create.js

```diff
@@ -4,15 +4,35 @@ import {Create as BaseCreate, SimpleForm} from 'admin-on-rest';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+const resolveProps = props => {
+  const {options} = props;
```

Comment:
`const {options: {inputFactory: defaultInputFactory, resource}} = props`?

Reply:
It is not possible because I use `options` in `return` (see line 21).